thijsvanloef/palworld-server-docker
# Add Docusaurus wiki

## Context 

Add Docusaurus to eventually replace the massive readme and make the information easily searchable.

## Choices

Docusaurus over Docsify as I'm more familiar with the capabilities

## Test instructions

Go to https://palworld-server-docker.loef.dev/ and see if there is any information/links missing

## Checklist before requesting a review
